File: desafios/fizzBuzz.js
fizzBuzz = () => {
  for (i = 1; i <= 100; i++) {
    console.log((i % 3 == 0 && i % 5 == 0) ? "FizzBuzz" : (i % 3 == 0) ? "Fizz" : (i % 5 == 0) ? "Buzz" : i);
  }
};

fizzBuzz(3);
fizzBuzz(5);
fizzBuzz(15);

File: Solucoes/fizzbuzz.cpp
#include <iostream>
using namespace std;

//https://dojopuzzles.com/problems/fizzbuzz/

//Declaração da função fizzbuzz
void fizzbuzz(){  
  for(int i = 1; i < 101; i++){
    if((i%3 == 0) && (i%5 == 0)){
      cout << "FizzBuzz";
    } else if(i%3 == 0){
      cout << "Fizz";  
    } else if(i%5 == 0){
      cout << "Buzz";  
    } else{
      cout << i ;
    }
    cout << "\n";
  }  
}

int main() {
  //Chamada da função
  fizzbuzz();  
}

